srv/salt/ceph/tuned/mgr/default.sls

```diff
@@ -0,0 +1,19 @@
+/usr/lib/tuned/ses-mgr/tuned.conf:
+  file.managed:
+    - source: salt://ceph/tuned/ses-mgr/tuned.conf
+    - makedirs: True
+    - user: root
+    - group: root
+    - mode: 644
+
+apply tuned ses mgr:
+  cmd.run:
+    - name: "tuned-adm profile ses-mgr"
+
+start-tuned:
+  cmd.run:
+    - name: "systemctl start tuned"
+
+enable-tuned:
+  cmd.run:
+    - name: "systemctl enable tuned"
```

Comment:
The start and enable can be collapsed into 

start tuned:
  service.running:
    - name: tuned
    - enable: True

unless that is specifically not behaving.

Reply:
No, you're right. Will update the PR.

Reply:
ups - this comment was already there ;-)